### PR TITLE
Record scenarios

### LINKED ISSUE #52: Flesh out multiple simulated scenarios vs one long one
Based on discussion in #34 we need to flesh out that our approach will use multiple simulated scenarios vs one long one because of potential issues at different absolute incidence levels and as it will be easier to communicate.

### PR DESCRIPTION
**Pull request by SamuelBrand1**
This mini-PR simply records the proposed scenarios in the manuscript.

Closes #52

### CODE REVIEW

File: manuscript/index.qmd [hunk 1]
```diff
@@ -78,19 +78,36 @@ We produce the simulations described in the next section for both of these GIs.
...
-    - 1.1 for six weeks
-    - sine curve centered at 1 with amplitude of 0.3 afterwards
+We test the following scenarios grouped by the **Outbreak**, **Endemic**, and **Miscellaneous** categories:
+
+**Outbreak scenarios**
+
+This is the list of scenarios where the initial number of infections is small but $R_t$ is initially significantly greater the 1 (e.g. $R_t > 1.5$).
+
+- _Susecptible depletion_. A smooth transition over time from $R_t > 1$ to $R_t < 0$. This represents a scenario where decrease in $R_t$ is due to greater population immunity, although it should be noted that we aren't modelling that effect mechanistically.
+- _Susecptible depletion with measures_. A sharp/discontinuous transition over time from $R_t > 1$ to $R_t < 0$, followed by a sharp transition back to $R_t > 1$ and then smooth transition to $R_t < 0$ again. This represents a scenario where initial decrease in $R_t$ is due to implementation of public health measures to reduce transmission. The sharp transition back to $R_t > 1$ is due to later relaxation of measures.
```
Comment (seabbs) [inline manuscript/index.qmd:88]: I think we want the sharp transition to be prior to R_t hitting 0

Comment (seabbs) [reply manuscript/index.qmd:88]: i.e If you mean Rt < 1 here that makes sense to me

File: manuscript/index.qmd [hunk 2]
```diff
@@ -78,19 +78,36 @@ We produce the simulations described in the next section for both of these GIs.
...
+- _Early outbreak_. Constant $R_t = R_0$ but for a short period.
+- _Early outbreak with random effects_. As _Early outbreak_ scenario but with white noise jitter on $R_t$.
+
+**Endemic scenarios**
+
+- _Regular variation_. A scenario with an endemic disease with sinusoidal variation in $R_t$ around 1 with some period length $P$: e.g. $R_t = 1 + \xi \sin(2 \pi (t - \phi) / P)$.
+- _Regular variation with random effects_. As _Regular variation_ scenario but with white noise jitter on $R_t$.
+
+**Miscellaneous scenario(s)**
```
Comment (seabbs) [inline manuscript/index.qmd:97]:
```suggestion
**Mixed scenario(s)**
```

**Review by seabbs — COMMENTED** [review]
This looks good with minor comments.

File: manuscript/index.qmd (hunk 1 above)
Comment (SamuelBrand1) [reply manuscript/index.qmd:88]: Yes. Typo

**Review by seabbs — APPROVED** [review]

**Pull request merged**